discordbot/stocks/technical_analysis/cci.py

```diff
@@ -41,13 +41,15 @@ async def cci_command(ctx, ticker="", length="14", scalar="0.015", start="", end
             end = datetime.now()
         else:
             end = datetime.strptime(end, cfg.DATE_FORMAT)
-
-        if not length.lstrip("-").isnumeric():
-            raise Exception("Number has to be an integer")
-        length = float(length)
-        if not scalar.lstrip("-").isnumeric():
-            raise Exception("Number has to be an integer")
-        scalar = float(scalar)
+        # pylint
+        try:
```

Comment:
I think that the previous solution is better than try except, but keep the int(lenght). Why did you just change it here to try except? The rest of the discord bot looks excellent!

Reply:
So the previous way was not working with the default strings.

I went with this quote just to only make 1 call from string to number.  I was also confused why there was a strip("-") in there.

Reply:
Didn't realise that it wasn't working before :)